src/site/content/en/reliable/http-cache/index.md

```diff
@@ -55,44 +80,76 @@ with specific
 overrides set. That's beyond the scope of this guide, though!
 {% endAside %}
 
-## Response headers: configure your web server
+## Response headers: configure your web server {: #response-headers }
 
 The part of the HTTP caching setup that matters the most is the headers that
-your web server adds to each outgoing response. A combination of the
-[`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control),
-[`ETag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag), and
-[`Last-Modified`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified)
-headers all factor into effective caching behavior.
+your web server adds to each outgoing response. The following headers all factor
+into effective caching behavior:
+
+* [`Cache-Control`][cache-control].
+  The server can return a `Cache-Control` directive to specify how, and for how
+  long, the browser and other intermediate caches should cache the individual
+  response.
+* [`ETag`][etag]. When
+  the browser finds an expired cached response, it can send a small token 
+  (usually a hash of the file's contents) to the server to check if the file has
+  changed. If the server returns the same token, then the file is the same, and there's
+  no need to re-download it.
+* [`Last-Modified`][last-modified].
```

Comment:
I think this language comes from https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified, but I'm not in favor of it. I'd prefer to think of it as there being two ways of the server identifying the version of an HTTP response: content-based (`ETag`) or time-based (`Last-Modified`).

If we're talking about, e.g., a response for an API feed that updates once every 15 minutes, then using `Last-Modified` and including the time that the API feed was last generated should be just as "accurate" as using an `ETag`. It's also conceptually more straightforward.

So, different options that each could perform a similar role.

Reply:
Yes, that is where I got that language. Your explanation makes sense. I'll re-word.